### PR TITLE
make predictors more discoverable

### PR DESCRIPTION
**Pull request by DeNeutoy**

### CODE REVIEW

**Comment by joelgrus** [comment]
👍 this is a good idea

**Comment by DeNeutoy** [comment]
Warnings look like this:
```
(allennlp) markn@markn ~/allen_ai/allennlp (expose-predictors) $ python
Python 3.6.3 |Anaconda, Inc.| (default, Oct  6 2017, 12:04:38)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import allennlp.service.predictors
/Users/markn/allen_ai/allennlp/allennlp/service/predictors/__init__.py:23: FutureWarning: allennlp.service.predictors.* has been depreciated. Please use allennlp.predictors.*
  "Please use allennlp.predictors.*", FutureWarning)
```

**Comment by DeNeutoy** [comment]
Diff looks large, but it's just shuffling the `predictors` around.

**Comment by matt-gardner** [comment]
Nit: "deprecated", not ["depreciated"](https://www.google.com/search?q=define%3Adepreciate).  Though the latter may also apply...

File: scripts/check_docs.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
         'allennlp.modules.alternating_highway_lstm',
         # Private base class, no docs needed.
         'allennlp.modules.encoder_base',
+        # Deprecated in favour of allennlp.predictors.
```
Comment (joelgrus) [inline scripts/check_docs.py]: favor 🇺🇸

**Pull request merged**